src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -746,6 +743,7 @@ def _get_node_referenced_flow_inputs(
                 node_referenced_flow_inputs[value.value] = flow_inputs[value.value]
         return node_referenced_flow_inputs
 
+    @entry_trace
```

Comment:
Let's do it like `@trace(type=TraceType.FLOW)`, and bind callbacks to the enum to edit the span attributes.